recipes/mysql-connector-cpp/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
The test_v1_package can be removed: https://github.com/conan-io/conan-center-index/discussions/17406